config/default/kustomization.yaml

```diff
@@ -25,7 +25,7 @@ resources:
 - ../namespace
 - ../crd
 - ../rbac
-- ../webhook
```

Comment:
can we remove rather than just comment out?

Reply:
I've left it here because it was originally generated by operator-sdk and other (commented out, autogenerated) places still reference it.